library/src/main/java/io/ona/kujaku/downloaders/MapBoxOfflineResourcesDownloader.java

```diff
@@ -281,16 +283,22 @@ public void onError(String error) {
      * @param name Unique name of the map
      * @param onDownloadMapListener Callback to give updates on download progress or errors
      */
-    public void resumeMapDownload(final String name, final OnDownloadMapListener onDownloadMapListener) {
+    public void resumeMapDownload(@NonNull final String name, final OnDownloadMapListener onDownloadMapListener) {
         if (offlineManager == null) {
-            onDownloadMapListener.onError("Context passed is null", "");
+            Log.e(TAG, "Context passed is null");
+            if (onDownloadMapListener != null) {
+                onDownloadMapListener.onError("Context passed is null", "");
```

Comment:
What's the difference between error reason and error message in the onDownloadMapListener. Should the error message in this onDownloadMapListener call be blank?

Reply:
A resumeMapDownload call. Above